### PR TITLE
the first commit

### PR DESCRIPTION
**Pull request by OlexSP**

### CODE REVIEW

File: first-go-lib.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Author: Alex Aloshchenko
+ * File: first-go-lib.go
+ */
+
+package lib
```
Comment (MrKAKTyC) [inline first-go-lib.go:9]: Just for more practice let's add a couple more functions like division and multiply

File: first-go-lib.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+// Add Returns the sum of two numbers
+func Add(a int, b int) int {
+	return a + b
+}
```
Comment (MrKAKTyC) [inline first-go-lib.go]: Let's add more sense to this function by adding handling integer overflow. Such cases like this `Add(math.MaxInt, 1)` will return an error instead negative value

File: first-go-lib.go (hunk 1 above)
Comment (MrKAKTyC) [inline first-go-lib.go:9]: Add overall package description

Comment (OlexSP) [reply first-go-lib.go:9]: ✅

Comment (OlexSP) [reply first-go-lib.go:9]: ✅

File: first-go-lib.go (hunk 2 above)
Comment (OlexSP) [reply first-go-lib.go]: ✅

**Comment by MrKAKTyC** [comment]
Approved

**Comment by MrKAKTyC** [comment]
Approved

**Pull request merged**